wordpress-mobile/WordPress-Android
# Fix blogging prompts select correct site in picker and minor UI fixes

Fixes #

1 - Fix blogging prompts onboarding selected site
To test:
- Enable blogging prompts feature flag
- Open blogging prompts onboarding from "My Site" card "Learn more" button or from "?" button in compact card
- Tap on "Remind me" button
- Select a site different than the one already selected on "My Site" tab
- Selected site on "My Site" tab should be updated matching the one selected on site picker AND reminder flow should be opened

2 - Remove click listener from "Answered" buttons
- Enable blogging prompts feature flag
- Make sure you've answered today's prompt
- Open "My Site" and tap on "HOME" tab
- Tap on "Answered" in the prompts card. Nothing should happen.
- Tap on the FAB
- Tap on "Answered" in the compact card. Nothing should happen.

3 - Fix prompt reminder notification check
- Enable blogging prompts feature flag
- On `My Site`, tap on `Menu`
- Tap on `Site Settings`
- Tap on `Reminders and prompts` and select a date and time for the reminder
- Change the device date to something after the selected date/time for the reminder
- A notification to answer the prompt should appear
- Tap on the notification and the editor should be opened
- Disable the feature flag and repeat the steps. The existing reminder notification should be shown.

4 - Fix reminder notification color icon for JP
- Run a Jetpack variant of the app and enable blogging prompts feature flag
- On `My Site`, tap on `Menu`
- Tap on `Site Settings`
- Tap on `Reminders and prompts` and select a date and time for the reminder
- Change the device date to something after the selected date/time for the reminder
- The notification icon should be green
- Disable the feature flag and repeat the steps. The notification icon should be green.
- Run a WordPress variant of the app and enable blogging prompts feature flag
- Repeat the steps above to display the reminder notification
- The notification icon should remain blue (same as it is today)
- Disable the feature flag and repeat the steps. The existing reminder notification should be blue.

## Regression Notes
1. Potential unintended areas of impact


2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
